### PR TITLE
Goodness: Alfred Workflow

### PR DESCRIPTION
**Pull request by yuriteixeira**
Alfred Workflow added.

### CODE REVIEW

**Review by nikitabobko — CHANGES_REQUESTED** [review]
Thanks!

1. Please specify the license in your git repo
2. Please specify how to build `.alfredworkflow` file from sources
3. In goodness.adoc, please mention that it's a third party extension (like it's done for raycast)
4. I want goodness.adoc page contain something that keeps the quality bar. Will you be willing to use `--json` to make your script more reliable? #577 (Once `--json` is implemented)

I'm curious why did you choose to focus between windows on the currently visible workspaces. Is it because alfread already allows to switch between all windows? 

Alternatively, You could also choose to switch between windows on the focused workspaces (the difference between focused workspace and visible workspaces is that, in multi-monitor setup, you have only one focused workspace, but as many visible as many monitors you have)

**Comment by nikitabobko** [comment]
Closing due to inactivity. Feel free to reopen PR, once you fix the comments

File: docs/goodness.adoc [hunk 1]
```diff
@@ -7,6 +7,15 @@ Do you have a cool automatization, AeroSpace integration, or workflow?
 Feel free to open an issue or pull request to add it to this list!
 The source code of the page can be found in the `./docs` directory.
 
+[#alfred-workflow]
+== Alfred Workflow
+
+[alfred,workflow,automation]
```
[the remark this answers is not in the document]
Comment (yuriteixeira) [reply docs/goodness.adoc:13]: Hello 👋 (and sorry for the delay). 

I copied/pasted an entry from another tool and those looked like tags or something, I can see if missed something.

**Comment by yuriteixeira** [comment]
> Thanks!
> 
> 1. Please specify the license in your git repo

Sure, will do.

> 2. Please specify how to build `.alfredworkflow` file from sources

Afaik, you need to do it using the Alfred UI

Screenshot 2024-10-24 at 09 28 38

> 3. In goodness.adoc, please mention that it's a third party extension (like it's done for raycast)

Sure, will do. 

> 4. I want goodness.adoc page contain something that keeps the quality bar. Will you be willing to use `--json` to make your script more reliable? [Add `--json` flag to all query commands #577](https://github.com/nikitabobko/AeroSpace/issues/577) (Once `--json` is implemented)

Sure! But just to clarify: did I miss it or wasn't it implemented?

> I'm curious why did you choose to focus between windows on the currently visible workspaces. Is it because alfread already allows to switch between all windows?
> 

Out of the box (without any 3rd party workflows), Alfred allows to switch between apps, but not windows.

> Alternatively, You could also choose to switch between windows on the focused workspaces (the difference between focused workspace and visible workspaces is that, in multi-monitor setup, you have only one focused workspace, but as many visible as many monitors you have)

The intention of my workflo was to allow the user to list all visible windows (amongst all visible workspaces).

**Comment by yuriteixeira** [comment]
Btw, it doesn't seem that I have permissions to re-open the PR. Maybe because I didn't push any changes? Anyways, I can ping the admin and ask them to re-open later.

**Pull request closed without merging**